scripts/build.sh

```diff
@@ -28,6 +28,7 @@ function build_archive {
 }
 
 function upload_assets {
+  # ${GH_TOKEN} is referred from the secure variable inside .travis.yml
```

Comment:
How about "referred from the secure variable inside .travis.yml" --> "defined as a secure variable in .travis.yml"?

Reply:
Also, I'd add a comment at the top of this file, saying something like:

    # This script is executed by Travis CI when a PR is merged (i.e. in the `after_success` step).

Reply:
Done